UnitTests/UnitTests.HighPerformance.Shared/Extensions/Test_ReadOnlySpanExtensions.cs

```diff
@@ -267,5 +267,35 @@ public void Test_ReadOnlySpanExtensions_Tokenize_CommaWithMissingValues()
 
             CollectionAssert.AreEqual(result, tokens);
         }
+
+        [TestCategory("ReadOnlySpanExtensions")]
+        [TestMethod]
+        public void Test_ReadOnlySpanExtensions_CopyTo_RefEnumerable()
+        {
+            int[,] array1 = new int[4, 5];
+
+            ReadOnlySpan<int> values = new[] { 10, 20, 30, 40, 50 };
+
+            // Copy a span to a target row and column with valid lengths
+            values.CopyTo(array1.GetRow(0));
+            values.Slice(0, 4).CopyTo(array1.GetColumn(1));
```

Comment:
Should we check the intermediary results? I've noticed these tests are chaining quite a few operations together, so it may be better to help find problems in a particular part if we're validating the first part before moving on? (Or split them as separate tests?)

Reply:
I think I addressed this in 6bb9e5a52e2fe199e0e2c2e2092183534a5a4ee8, let me know if that's alright 😄